ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -13900,7 +13901,11 @@ private void addDbAndTabToOutputs(String[] qualifiedTabName, TableType type,
     for(Map.Entry<String,String> serdeMap : storageFormat.getSerdeProps().entrySet()){
       t.setSerdeParam(serdeMap.getKey(), serdeMap.getValue());
     }
-    outputs.add(new WriteEntity(t, WriteEntity.WriteType.DDL_NO_LOCK));
+    if (tblProps.get("created_with_ctas") == "true") {
```

Comment:
is there any constant for "created_with_ctas"?

Reply:
Thanks for the review:) .
 Yes, will update this.